apps/hash-api/src/graphql/resolvers/knowledge/entity/entity.ts

```diff
@@ -204,23 +196,21 @@ export const getEntityResolver: ResolverFn<
       hasLeftEntity,
       hasRightEntity,
     },
-    temporalAxes: {
-      pinned: {
-        axis: "transactionTime",
-        timestamp: null,
-      },
-      variable: {
-        axis: "decisionTime",
-        interval: {
-          start: entityVersion
-            ? { kind: "inclusive", limit: entityVersion }
-            : null,
-          end: entityVersion
-            ? { kind: "inclusive", limit: entityVersion }
-            : null,
-        },
-      },
-    },
+    temporalAxes: entityVersion
+      ? {
+          pinned: {
+            axis: "transactionTime",
+            timestamp: null,
+          },
+          variable: {
```

Comment:
I know you didn't add this, but maybe we want to add a comment saying "Constrain the results so that they overlap the given instant specified by the entityVersion timestamp" or something

Reply:
I moved that statement to a variable to make it more readable and added a comment on it: [Add comment to entity resolver](https://github.com/hashintel/hash/pull/2542/commits/663a1d1f94b400e672470c2228458dada5ceb401)